api/logging_api/routes/Printer.js

```diff
@@ -0,0 +1,70 @@
+const express = require('express');
+const fs = require('fs');
+const router = express.Router();
+const {OK} = require('../../util/constants').STATUS_CODES;
+const app = express();
+const s3BucketKeys = require('../../config/config.json').S3Bucket;
+const printingS3Bucket = require('../../config/config.json').PrintingS3Bucket;
+const queueKeys = require('../../config/config.json').Queue;
+
+// Load the AWS SDK for Node.js
+const AWS = require('aws-sdk');
+// Set the region we will be using
+let creds = new
+AWS.Credentials(s3BucketKeys.AWSACCESSKEYID, s3BucketKeys.AWSSECRETKEY);
+
+AWS.config.update({
+  region: 'us-west-1',
+  endpoint: 'https://s3.amazonaws.com',
+  credentials: creds
+});
+
+router.get('/healthCheck', (req, res) => {
+  // console.log(`Hello ${req.query.name}!`);
+  res.sendStatus(OK);
+});
+app.use(express.json());
```

Comment:
i don't think you need this. if the requests still work the same without it remove

Reply:
Works.